python/revng/model/v1/__init__.py

```diff
@@ -5,13 +5,15 @@
 from .._util import force_constructor_kwarg, enum_value_to_index
 
 _types_to_kinds = [
-    (PrimitiveType, TypeKind.Primitive),
-    (EnumType, TypeKind.Enum),
-    (TypedefType, TypeKind.Typedef),
-    (StructType, TypeKind.Struct),
-    (UnionType, TypeKind.Union),
+    (PrimitiveType, TypeKind.PrimitiveType),
+    (EnumType, TypeKind.EnumType),
+    (TypedefType, TypeKind.TypedefType),
+    (StructType, TypeKind.StructType),
+    (UnionType, TypeKind.UnionType),
     (CABIFunctionType, TypeKind.CABIFunctionType),
     (RawFunctionType, TypeKind.RawFunctionType),
+    (FunctionEdge, FunctionEdgeBaseKind.FunctionEdge),
+    (CallEdge, FunctionEdgeBaseKind.CallEdge),
 ]
 for t, k in _types_to_kinds:
     force_constructor_kwarg(t, "Kind", k)
```

Comment:
All of this can be reworked to be done directly in generated code now.
Take a look and if you have doubts ask @fcremo.